test/rule_test.dart

```diff
@@ -31,17 +31,28 @@ main() {
 }
 
 final String ruleDir = p.join('test', 'rules');
+final String testConfigDir = p.join('test', 'configs');
 
 /// Rule tests
 defineRuleTests() {
   // TODO: if ruleDir cannot be found print message to set CWD to project root
   group('rule', () {
     group('dart', () {
-      for (var entry in new Directory(ruleDir).listSync()) {
-        if (entry is! File || !isDartFile(entry)) continue;
-        var ruleName = p.basenameWithoutExtension(entry.path);
-        testRule(ruleName, entry, debug: true);
-      }
+      // Rule tests run with default analysis options.
+      testRules(ruleDir, analysisOptions: null);
```

Comment:
When we enable the code below we might want to wrap this in a group titled '(default)'.

Reply:
Sure.